ep/build-velox/src/get_velox.sh

```diff
@@ -107,7 +107,7 @@ function process_setup_centos9 {
   sed -i '/^.*dnf_install autoconf/a\  dnf_install libxml2-devel libgsasl-devel libuuid-devel' scripts/setup-centos9.sh
   
   ensure_pattern_matched 'install_gflags' scripts/setup-centos9.sh
-  sed -i '/^function install_gflags.*/i function install_openssl {\n  wget_and_untar https://github.com/openssl/openssl/archive/refs/tags/OpenSSL_1_1_1s.tar.gz openssl \n  cd openssl \n  ./config no-shared && make depend && make && sudo make install \n  cd ..\n}\n'     scripts/setup-centos9.sh
+  sed -i '/^function install_gflags.*/i function install_openssl {\n  wget_and_untar https://github.com/openssl/openssl/releases/download/openssl-3.2.2/openssl-3.2.2.tar.gz openssl \n  cd openssl \n  ./config no-shared && make depend && make && sudo make install \n  cd ..\n}\n'     scripts/setup-centos9.sh
```

Comment:
should use `cd ${DEPENDENCY_DIR}/openssl` as recent Velox have this change?

Reply:
Velox doesn't seem to be bringing openssl 3.x  as a part of [centOS9 setup](https://github.com/oap-project/velox/blob/17ec2bb2d80f7db4093ca33546dbcb37d8ec9748/scripts/setup-centos9.sh).

Reply:
@pratham76 `scripts/setup-centos9.sh` is coming from Velox.
In Velox, there was a recent change https://github.com/facebookincubator/velox/pull/10920 to use `${DEPENDENCY_DIR}` for dependencies. As @zhouyuan suggested, we need to use `cd ${DEPENDENCY_DIR}/openssl` when `wget_and_untar` is used.
Can you validate your change on the latest gluten main branch?

Reply:
got it. will update and validate.
This change does not seem to be added to gluten main branch yet - there are other dependencies being installed in current directory.

Reply:
Interesting! Velox is updated daily and I do see this PR here https://github.com/oap-project/velox

Reply:
yes but gluten overrides those and adds new installs in this [script](https://github.com/apache/incubator-gluten/blob/main/ep/build-velox/src/get_velox.sh) which also has to be updated.

Reply:
Then why is ${DEPENDENCY_DIR} not being used?

Reply:
Have updated the changes as requested and validated the build with manual tests.  Kindly review the changes. In the above comment i was noting that as gluten overrides the velox setup scripts using `sed` changes wrt install directories have to be added here too.